assets/src/setup/pages/save/index.js

```diff
@@ -41,16 +106,84 @@ export function Save() {
 	}, [ didSaveDeveloperToolsOption, savingDeveloperToolsOption, saveDeveloperToolsOption ] );
 
 	if ( savingOptions || savingDeveloperToolsOption ) {
-		return <Loading />;
+		return <Saving />;
 	}
 
-	if ( ! didSaveOptions || ! didSaveDeveloperToolsOption ) {
-		return null;
+	let heading = __( 'Congratulations!', 'amp' );
+	if ( 'standard' === themeSupport ) {
+		heading = __( 'Your site is ready', 'amp' );
 	}
 
 	return (
-		<div>
-			{ __( 'Options saved', 'amp' ) }
+		<div className="done grid grid-6-5">
+			<div>
+				<h1>
+					{ heading }
+				</h1>
+				<p>
+					{ getDescription( themeSupport ) }
+				</p>
+			</div>
+			<div className="done__preview-container">
+				<AMPInfo icon={ ( props ) => <IconMobile { ...props } /> }>
+					{ __( 'Live view of your site', 'amp' ) }
+				</AMPInfo>
+				<Phone>
+					<iframe
+						className="done__preview-iframe"
+						src={ addQueryArgs( PREVIEW_PERMALINK, 'standard' === themeSupport ? {} : { amp: 1 } ) }
+						title={ __( 'Site preview', 'amp' ) }
+						name="amp-wizard-completion-preview"
+					/>
+				</Phone>
+
+				{ 'reader' !== themeSupport && (
+					<Button
+						isPrimary
+						href={
+							addQueryArgs(
+								PREVIEW_PERMALINK,
+								'transitional' === themeSupport
+									? { amp: 1 }
+									: {},
+							)
+						}
+						target="_blank"
+						rel="noreferrer"
+					>
+						{ __( 'Visit your site', 'amp' ) }
+					</Button>
+				) }
+
+				{
+					'reader' === themeSupport && (
+						<>
+							<Button
+								isPrimary
+								href={
+									addQueryArgs(
+										CUSTOMIZER_LINK,
+										'legacy' === readerTheme
+											? { 'autofocus[panel]': 'amp_panel', url: PREVIEW_PERMALINK }
+											: { url: PREVIEW_PERMALINK },
```

Comment:
The `amp` query var needs to be added when going to the non-Legacy Reader mode Customizer:

```suggestion
											: { url: PREVIEW_PERMALINK, amp: 1 },
```

Reply:
I suppose this won't be needed with the preview permalink returned via the Options endpoint.

Reply:
It will because this is an `amp` query var for `customize.php` not the `PREVIEW_PERMALINK`.

Reply:
Granted, at the moment this does nothing. But it _will_.

Reply:
OK, I hadn't picked up on the customizer link also needing the `amp` query var. Updated.